lightning/src/ln/channelmanager.rs

```diff
@@ -6881,13 +6973,17 @@ where
 			best_block.block_hash().write(writer)?;
 		}
 
+		let mut peers_to_searialize_count: u64 = 0;
```

Comment:
nit: `serialize`. Though a simple `peer_count` or `serializable_peer_count` should suffice.